Gemfile

```diff
@@ -73,6 +73,7 @@ gem "synaccess_connect", "0.2.2", github: "tablexi/synaccess"
 
 group :development do
   gem "coffeelint"
+  gem "haml_lint", github: "brigade/haml-lint", ref: "1384ec"
```

Comment:
What's the reason for using the github ref instead of the released version of the gem?